jbpm-human-task/jbpm-human-task-workitems/src/main/java/org/jbpm/services/task/wih/ExternalTaskEventListener.java

```diff
@@ -105,7 +106,7 @@ public void afterTaskCompletedEvent(TaskEvent event) {
             throw new RuntimeException("No RuntimeManager registered with identifier: " + task.getTaskData().getDeploymentId());
         }
         RuntimeEngine runtime = manager.getRuntimeEngine(ProcessInstanceIdContext.get(processInstanceId));
-        KieSession session = runtime.getKieSession();
+        KieSession session = ((InternalRuntimeEngine) runtime).internalGetKieSession();
```

Comment:
Just if I get it right - this swap of methods allows us to omit the call to `((AbstractRuntimeManager) manager).checkPermission();`?